projects/angular/src/accordion/stepper/stepper-panel.spec.ts

```diff
@@ -88,13 +88,13 @@ describe('ClrStep Reactive Forms', () => {
       fixture.detectChanges();
 
       const statusMessage = fixture.nativeElement.querySelector('button .clr-sr-only');
-      expect(statusMessage.innerText.trim()).toBe('Error');
+      expect(statusMessage.innerText.trim()).toBe(`Error in step ${mockStep?.index + 1}`);
```

Comment:
The expected value should not be computed at runtime. Please replace the template literal with a static string literal. This applies to both tests.

Reply:
Made these changes

Reply:
Thanks!